### DISCUSSION TITLE
Transferring Computers

### DISCUSSION THREAD

**Discussion by Covakel** [Courseplay/Courseplay_FS22#1138] [Q&A] [answered]
Good Day, just to start, this is an awesome mod and you guys have done amazing work!

I play FS22 on two different computers as I put it on my laptop when I'm travelling. If I'm moving to another computer, what files/folder do I need to copy so I have all my CoursePlay routes on the other computer?

I tried it once before copying over my SaveGame1 folder, but it didn't copy the CoursePlay routes as I think they're stored in the mapUS folder somewhere.

Many thanks!

**Comment by Tensuko** [comment, marked as answer]
...\My Games\FarmingSimulator2022\modSettings\FS22_Courseplay

If you travel with mods, you should also copy your modSettings Folder, but for Courseplay, that Folder is needed for some stuff.

**Reply by Covakel** [reply]
Excellent, thank you!
